src/sentry/static/sentry/app/views/organizationHealth/eventsTableChart.jsx

```diff
@@ -0,0 +1,139 @@
+import {Flex} from 'grid-emotion';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import Count from 'app/components/count';
+import InlineSvg from 'app/components/inlineSvg';
+import TableChart from 'app/components/charts/tableChart';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import space from 'app/styles/space';
+
+const Delta = ({current, previous, className}) => {
+  const changePercent = Math.round(Math.abs(current - previous) / previous * 100);
+  const direction = !changePercent ? 0 : current - previous;
+  return (
+    <StyledDelta direction={direction} className={className}>
+      {!!direction && <DeltaCaret direction={direction} src="icon-chevron-down" />}
+      {changePercent !== 0 ? `${changePercent}%` : <span>&mdash;</span>}
+    </StyledDelta>
+  );
+};
+Delta.propTypes = {
+  current: PropTypes.number,
+  previous: PropTypes.number,
+};
+
+const DeltaCaret = styled(InlineSvg)`
+  /* should probably have a chevron-up svg (: */
+  ${p => p.direction > 0 && 'transform: rotate(180deg)'};
+  width: 10px;
+  height: 10px;
+`;
+
+const StyledDelta = styled(Flex)`
+  align-items: center;
+  padding: 0 ${space(0.25)};
+  margin-right: ${space(0.5)};
+  font-size: ${p => p.theme.fontSizeSmall};
+  color: ${p =>
+    p.direction > 0 ? p.theme.green : p.direction < 0 ? p.theme.red : p.theme.gray2};
+`;
+
+class EventsTableChart extends React.Component {
+  static propTypes = {
+    headers: PropTypes.arrayOf(PropTypes.node),
+    data: PropTypes.arrayOf(
+      PropTypes.shape({
+        name: PropTypes.node,
+        percentage: PropTypes.number,
+        count: PropTypes.number,
+        lastCount: PropTypes.number,
+      })
+    ),
+  };
+
+  render() {
+    const {headers, data} = this.props;
+
+    return (
+      <TableChart
+        headers={headers}
+        data={data.map(({count, lastCount, name, percentage}) => [
+          <Name key="name">{name}</Name>,
+          <Events key="events">
+            <Delta current={count} previous={lastCount} />
+            <Count value={count} />
+          </Events>,
+          <React.Fragment key="bar">
```

Comment:
Fragment because we're wrapping 2 sibling react elements.

key props are because of eslint thinking we require them because these react elements are in an array.